static/src/js/components/AppLogo/AppLogo.scss

```diff
@@ -60,10 +60,10 @@
   &__site-name {
     display: inline-block;
     color: $color__white;
-    font-size: 1.125rem;
+    font-size: 1rem;
     text-transform: uppercase;
-    letter-spacing: 0.05rem;
     font-weight: 300;
+    font-family: "Inter";
```

Comment:
This should probably have some fallback to the system font stack

Reply:
This now uses the same overridden bootstrap variable so it falls back to those fonts if Inter isn't available.